### PR TITLE
Update template.md

### PR DESCRIPTION
**Pull request by badersur**

### CODE REVIEW

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  No file of this docset changes.  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C19%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609191103145682-1033%5Cworkflow_report.html).

File: styleguide/template.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ The full metadata block is above (in the [raw Markdown](https://raw.githubuserco
 - Colons in a value (for example, a title) break the metadata parser. In this case, surround the title with double quotes (for example, `title: "Writing .NET Core console apps: An advanced step-by-step guide"`).
 - **title**: This title will appear in search engine results. You can also add a pipe (|) followed by the product name (for example, `title: Developing Libraries with Cross Platform Tools | .NET Core`). The title doesn't need be identical to the title in your H1 heading and it should contain 65 characters or less (including | PRODUCT NAME).
 - **author**, **manager**, **ms.reviewer**: The author field should contain the **GitHub username** of the author, not their alias.  The "manager" and "ms.reviewer" fields, on the other hand, should contain Microsoft aliases. ms.reviewer specifies the name of the PM/dev associated with the article or feature.
-- **ms.devlang** defines the technology. Some of the supported values are: dotnet, cpp, csharp, fsharp, xml, vb and xml.
```
Comment (badersur) [inline styleguide/template.md]: maybe you support xaml as well as xml?!

Comment (mairaw) [inline styleguide/template.md]: @badersur xaml is not one of the supported values for this field. It was probably an oversight to have xml twice.

**Review by mairaw — APPROVED** [review]
LGTM. Thanks for your contribution @badersur!

**Comment by badersur** [comment]
You are welcome @mairaw :)

**Pull request merged**